enhancements/cluster-logging/forwarder-input-selectors.md

```diff
@@ -117,37 +117,44 @@ Administrators create an instance of **ClusterLogForwarder** which defines which
 
 
 Following are the additions to the InputSpec:
+
 * Application Input
 ```yaml
     spec:
     - name: my-app
       application:
-        namespaces: []           #exact string or glob
-        excludeNamespaces: []    #exact string or glob
+        namespaces: []           #deprecated: exact string or glob
+        includes:
+        - container:             #exact string or glob
+          namespace:             #exact string or glob
+        excludes:
+        - container:             #exact string or glob
+          namespace:             #exact string or glob
         selector:                #metav1.LabelSelector
           matchLabels: []
           matchExpressions:
           - key:
             operator:
             values: []
-        containers:
-          include: []            #exact string or glob
-          exclude: []            #exact string or glob
 ```
+
+**NOTE:** *application.namespaces* field is deprecated.
+
 ```golang
    type Application struct {
      Namespaces        []string
-     ExcludeNamespaces []string
+     Includes          []string
```

Comment:
Shouldn't this also be a `*NamespaceContainerGlob`? Or is this just going to be a `[]string`?